example/CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to AWS ARC Network
+Thank you for considering contributing to AWS ARC Network! We appreciate your time and effort.
+To ensure a smooth collaboration, please take a moment to review the following guidelines.
+
+## How to Contribute
+1. Fork the repository to your own GitHub account.
+2. Clone the repository to your local machine.
+   ```bash
+   git clone https://github.com/<your_organization>/<your_terraform_module>.git
```

Comment:
why you have added the same file in example folder? remove it

Reply:
done